portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,22 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("hi my name is </br> ashley granitto");
+    // Initial list of nametag greetings
+    ArrayList<String> greetings = new ArrayList<String>();
```

Comment:
You should be able to leave off the generic argument in the constructor, as it can infer the type

```java
ArrayList<String> greetings = new ArrayList<>();
```

https://docs.oracle.com/javase/8/docs/technotes/guides/language/type-inference-generic-instance-creation.html

Reply:
fixed